eng/versioning.targets

```diff
@@ -156,12 +156,10 @@
 
   </Target>
 
-  <!-- Removes specified assembly level attributes. https://github.com/dotnet/runtime/issues/44257-->
-  <Target Name="RemoveSupportedPlatformAssemblyLevelAttribute"
-          AfterTargets="GetAssemblyAttributes">
+  <!-- Removes assembly level attributes for Browser. -->
+  <Target Name="RemoveSupportedPlatformAssemblyAttributeForBrowser" AfterTargets="GetAssemblyAttributes" Condition="'$(TargetFrameworkSuffix)' == 'Browser'">
```

Comment:
We will remove this rule completely after separating `Unsupported on browser` APIs from browser build.